adhocteam/Head-Start-TTADP
# Enable multi role for users in admin

**Description of change**
Adds an ability to assign multiple roles to users in the admin UI.

**How to test**
- pull changes
- assign multiple roles to couple of users in the admin UI
- create reports
- verify that the roles are displayed on the landing page

**Issue(s)**
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-69

**Checklist**

- [n/a] Documentation updated
    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
